### DISCUSSION TITLE
A new fresh realm.

### DISCUSSION THREAD

**Discussion by V-i-c-t-o-r-2-5** [chromiecraft/chromiecraft#6620] [Suggestions]
Hello fellow time travelers.

I have a proposal that I consider interesting: a new realm, with the same concept as the previous one, an ultra-progression, but much faster. This would increase the popularity of the server, and with it the financial income to be able to maintain it, as well as an adventure for new or old people who want to start but from scratch.

**Comment by heyitsbench** [comment]
I think this is a good idea once the progression of the main realm is complete, otherwise it could easily cannibalize the small playerbase of the main realm.

**Reply by zenoslime** [reply]
i wonder. speaking for myself, i would 100% play on both. a cool idea imo: new realm locked at vanilla (but progressive), either permanently or behind the existing realm by one full expansion. hell, if at some point chromie had 3 servers -- one vanilla, one tbc, and one wrath -- i would play on all of them.

**Reply by tbcxd** [reply]
This new second realm would have 50 max players and all of them from the "main realm".
A new realm would only split the population.
Rather focus on a single realm, it's not like we get lag spikes from stormwind being too populated.

**Comment by elthehablo** [comment]
This is indeed something we will look at when nearing the end of progression in wotlk
